backend/src/index.ts

```diff
@@ -1,17 +1,13 @@
-import express from "express";
+import http from 'http';
+import app from './app';
 
 const PORT = process.env.PORT || 3030;
 
-const app = express();
+const server = http.createServer(app);
 
-const magicNumber = 42;
+if (process.env.NODE_ENV !== 'test')
+    server.listen(PORT, () =>
+        console.log(`server running on http://localhost:${PORT}`)
+    );
 
-app.get("/", (req, res) => {
-  res.send("Hello World");
-});
-
-const server = app.listen(PORT, () =>
-  console.log(`server running on http://localhost:${PORT}`)
-);
-
-export { server, magicNumber };
+export default server;
```

Comment:
export default is a bad practise, I'd recommend using named exports always.

Reply:
Fixed by [953fcd4](https://github.com/bytecraftoy/aalto-2022/pull/7/commits/953fcd4c6784914ac9664e268091e2d4bd037fad)